nav2_behaviors/include/nav2_behaviors/timed_behavior.hpp

```diff
@@ -259,6 +250,15 @@ class TimedBehavior : public nav2_core::Behavior
         return;
       }
 
+      if (action_server_->is_cancel_requested()) {
```

Comment:
Can you check the other Nav2 action servers and see if any others should be flipped as well?

Reply:
Everything else looks fine.